src/html2md.js

```diff
@@ -84,13 +89,54 @@ function assertValidJSON(str) {
   }
 }
 
+/**
+ * @param {string} str
+ * @param {number} [limit]
+ * @returns {string}
+ * @throws {Error}
+ */
 function assertMetaSizeLimit(str, limit = 128_000) {
   if (str && str.length > limit) {
     throw Error('metadata size limit exceeded');
   }
   return str;
 }
 
+/**
+ * Check if meta name is allowed:
+ *  - non-reserved
+ *  - not starting with 'twitter:'
+ *    - except 'twitter:label' and 'twitter:data'
+ * @param {string} name
+ * @returns {boolean}
+ */
+function isAllowedMetaName(name) {
+  if (typeof name !== 'string') {
+    return false;
+  }
+  return !HELIX_META[name] && (
+    !name.startsWith('twitter:')
+    || name === 'twitter:card'
+    || name.startsWith('twitter:label')
+    || name.startsWith('twitter:data')
+  );
+}
+
+/**
+ * Check if meta property is allowed:
+ *  - non-reserved
+ *  - og:type
+ *  - product:*
```

Comment:
the logic in html-pipeline is different:
```
but the logic in html-pipeline is different:

```
    const attr = name.includes(':') && !name.startsWith('twitter:') ? 'property' : 'name';
```

if use `property` if:
- name includes a `:`, but not starts with `twitter:`

Reply:
this is also consistent with the change